src/App.tsx

```diff
@@ -1,9 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setQuery(value);
+  };
+
+  const visibleMovies = moviesFromServer.filter(film => {
+    const { title, description } = film;
+
+    const searchByTitle = title
```

Comment:
if some code writen twice it can be placed in one function.
Create function for your  `.toLowerCase().includes(query.toLowerCase())`